tools/topology/topology2/development/tplg-targets.cmake

```diff
@@ -37,6 +37,9 @@ DEEPBUFFER_D0I3_COMPATIBLE=true"
 "cavs-nocodec-multicore\;sof-mtl-nocodec-multicore\;PLATFORM=mtl,SSP1_ENABLED=true,SSP0_CORE_ID=0,\
 SSP1_CORE_ID=1,SSP2_CORE_ID=2,PREPROCESS_PLUGINS=nhlt,NHLT_BIN=nhlt-sof-mtl-nocodec.bin"
 
+"cavs-nocodec-multicore\;sof-mtl-nocodec-multicore-ssp0-ssp2\;PLATFORM=mtl,SSP1_ENABLED=false,\
+SSP0_CORE_ID=0,SSP2_CORE_ID=1,PREPROCESS_PLUGINS=nhlt,NHLT_BIN=nhlt-sof-mtl-nocodec.bin"
```

Comment:
nit-pick: SSP0_CORE_ID is 0 by default. so no need to set it again here

Reply:
> nit-pick: SSP0_CORE_ID is 0 by default. so no need to set it again here
@ranj063 Thank you! I've removed this.